test/index.js

```diff
@@ -36,9 +36,9 @@ describe('components', () => {
   transition-timing-function: ease-out;
   -webkit-transition-duration: .3s;
   transition-duration: .3s;
-  -webkit-transform: translateX(NaN%);
-  -ms-transform: translateX(NaN%);
-  transform: translateX(NaN%);
```

Comment:
whoops!

Reply:
ìndex` was missing inside the Carousel test.